controllers/fleetsyncs/pkg/controllers/fleetsync/controller.go

```diff
@@ -713,6 +713,6 @@ func (r *FleetSyncReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Owns(&v1alpha1.FleetMembership{}).
 		Owns(&v1alpha1.FleetScope{}).
 		Owns(&v1alpha1.FleetMembershipBinding{}).
-		WatchesRawSource(&source.Channel{Source: channel}, &handler.EnqueueRequestForObject{}).
+		//WatchesRawSource(&source.Channel{Source: channel}, &handler.EnqueueRequestForObject{}).
```

Comment:
Why? This make the Fleetsync controller unisable I think.

Reply:
Similarly, with the upgrade to the controller-runtime libs, I was unsure of the fix here.
https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.18.5/pkg/builder#Builder.WatchesRawSource

The fleetsync stuff is GCP specific and I made a move to remove it but was declined by John B.
We should look to move it out somewhere though.

Reply:
If the request from John B. is that we never break this, then we should have the move out before we merge this update, right?

Reply:
I would tend to agree, but, I'm not sure this has even been tested/deployed since being ported to nephio. The gcp distro pkg still points to the kpt image - https://github.com/nephio-project/catalog/blob/main/distros/gcp/nephio-mgmt/porch/9-controllers.yaml#L46

Reply:
Fleetsync controller moved to here - https://github.com/nephio-project/porch-fleets
Next PR will remove this from porch repo.